Test/Utils/ContrailAPI/Global.ps1

```diff
@@ -0,0 +1,16 @@
+function FQNameToUuid {
+    Param ([Parameter(Mandatory = $true)] [string] $ContrailUrl,
+           [Parameter(Mandatory = $true)] [string] $AuthToken,
+           [Parameter(Mandatory = $true)] [string] $Type,
+           [Parameter(Mandatory = $true)] [string[]] $FQName)
+
+    $Request = @{
+        type     = $Type
+        fq_name  = $FQName
+    }
+
+    $FQToUuidUrl = $ContrailUrl + "/fqname-to-id"
+    $Response = Invoke-RestMethod -Uri $FQToUuidUrl -Headers @{"X-Auth-Token" = $AuthToken} `
+        -Method Post -ContentType "application/json" -Body (ConvertTo-Json -Depth $CONVERT_TO_JSON_MAX_DEPTH $Request)
```

Comment:
https://github.com/krdln/shelly would complain that `$CONVERT_TO_JSON_MAX_DEPTH` is an indirect import. Not that important, but just wanted to plug this awesome tool :)

Reply:
That is strange, because I directly imported it at the the first line of file. Maybe it is because another imported file also imports it and overrides the value.